### DISCUSSION TITLE
Multiple authentication methods enabled

### DISCUSSION THREAD

**Discussion by enribar** [apache/superset#18742] [Q&A / Help] [answered]
Hi, is it possible to have multiple authentication methods enabled at the same time? I noticed that if I enable AUTH_TYPE = AUTH_LDAP, DB authentication does not work anymore.

**Comment by srinify** [comment]
Hi there @xjobex we're trying to keep Issues focused on bugs! I've moved this to a Discussion instead :]

**Comment by enribar** [comment, marked as answer]
I asked on Flask-AppBuilder and they answered me that it is not supported: https://github.com/dpgaspar/Flask-AppBuilder/issues/1803#issuecomment-1040275350
